src/Json.php

```diff
@@ -57,27 +57,27 @@ public function getEndpoint()
         return $this->endpoint;
     }
 
-    public function get($path, array $params = [])
+    public function get($path, array $params = [], array $headers = [])
```

Comment:
I'm not sure how many other options are typically passed and if we're not making people type more than necessary, but sure! I don't object. I'll fiddle with it.